modules/subnet_router/container_definitions/tailscale.json

```diff
@@ -7,6 +7,10 @@
         {
             "name": "TAILSCALE_ADVERTISE_ROUTES",
             "value": "${advertise_routes}"
+        },
+        {
+            "name": "TAILSCALE_ADDITIONAL_FLAGS",
+            "value": "${additional_flags}"
```

Comment:
I wish it were a little more possible to say "`additional_flags` is null? don't pass the `TAILSCALE_ADDITIONAL_FLAGS` env. var."

However, I think the amount of code/config to get to that point isn't worth it, so fine to leave as is.

(PS I realize you made `""`, not `null`, as the default.)